### PR TITLE
New version: Evernote.Evernote version 10.101.6

### PR DESCRIPTION
**Pull request by spectopo**
Created by [🥟 Dumplings](https://github.com/SpecterShell/Dumplings) in workflow run [#7351](https://github.com/SpecterShell/Dumplings/actions/runs/10400804154).

<details>

<summary>Log</summary>

````
Updated: 10.101.5 → 10.101.6

<Error>
  <Code>NoSuchKey</Code>
  <Message>The specified key does not exist.</Message>
</Error>
Submitting WinGet manifests
Creating manifests
Uploading manifests and making commits
Creating a pull request
````

</details>

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/167809)

### CODE REVIEW

File: manifests/e/Evernote/Evernote/10.101.6/Evernote.Evernote.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.4.1 Dumplings Mod $debug=QUSU.CRLF.7-4-4.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Evernote.Evernote
+PackageVersion: 10.101.6
+InstallerType: nullsoft
+Scope: user
```
Comment (mdanish-kh) [inline manifests/e/Evernote/Evernote/10.101.6/Evernote.Evernote.installer.yaml:7]:
```suggestion
Scope: machine
```

Installation appears to be machine wide

image

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

File: manifests/e/Evernote/Evernote/10.101.6/Evernote.Evernote.installer.yaml (hunk 1 above)
Comment (SpecterShell) [reply manifests/e/Evernote/Evernote/10.101.6/Evernote.Evernote.installer.yaml]: Both scopes available. When the installer is run under admin, the machine-wide option will show up and become the default.

**Review by mdanish-kh — COMMENTED** [review]
@SpecterShell - little confused by the changes in [`b69252a` (#167809)](https://github.com/microsoft/winget-pkgs/pull/167809/commits/b69252ad0cbbbb67e58c47ef270db4fa035cd653). Looks to me we're doing user scope install on x86 machines, and machine scope install on x64 (and scope parameter also missing at installer level)? Shouldn't we do something like:

```
Installers:
- Architecture: x86
  InstallerUrl: https://win.desktop.evernote.com/builds/Evernote-10.101.6-win-ddl-stage-20240814195043-630c415c3da5ff7ae8f733a5c0d36a1691c06135-setup.exe
  InstallerSha256: 871368AB87E5DAEFA014CB5DF6BB7B95FBB5BA7FC505102B6B23F6D90198F701
  Scope: user
  InstallerSwitches:
    Custom: /currentuser
  ProductCode: e4251011-875e-51f3-a464-121adaff5aaa
- Architecture: x86
  InstallerUrl: https://win.desktop.evernote.com/builds/Evernote-10.101.6-win-ddl-stage-20240814195043-630c415c3da5ff7ae8f733a5c0d36a1691c06135-setup.exe
  InstallerSha256: 871368AB87E5DAEFA014CB5DF6BB7B95FBB5BA7FC505102B6B23F6D90198F701
  Scope: machine
  InstallerSwitches:
    Custom: /allusers
  ProductCode: e4251011-875e-51f3-a464-121adaff5aaa
  ElevationRequirement: elevationRequired
- Architecture: x64
  InstallerUrl: https://win.desktop.evernote.com/builds/Evernote-10.101.6-win-ddl-stage-20240814195043-630c415c3da5ff7ae8f733a5c0d36a1691c06135-setup.exe
  InstallerSha256: 871368AB87E5DAEFA014CB5DF6BB7B95FBB5BA7FC505102B6B23F6D90198F701
  Scope: user
  InstallerSwitches:
    Custom: /currentuser
  ProductCode: e4251011-875e-51f3-a464-121adaff5aaa
- Architecture: x64
  InstallerUrl: https://win.desktop.evernote.com/builds/Evernote-10.101.6-win-ddl-stage-20240814195043-630c415c3da5ff7ae8f733a5c0d36a1691c06135-setup.exe
  InstallerSha256: 871368AB87E5DAEFA014CB5DF6BB7B95FBB5BA7FC505102B6B23F6D90198F701
  Scope: machine
  InstallerSwitches:
    Custom: /allusers
  ProductCode: e4251011-875e-51f3-a464-121adaff5aaa
  ElevationRequirement: elevationRequired
```

**Comment by mdanish-kh** [comment]
[comment]: <[Policy] Needs-Author-Feedback>

**Comment by SpecterShell** [comment]
> @SpecterShell - little confused by the changes in [`b69252a` (#167809)](https://github.com/microsoft/winget-pkgs/pull/167809/commits/b69252ad0cbbbb67e58c47ef270db4fa035cd653). Looks to me we're doing user scope install on x86 machines, and machine scope install on x64 (and scope parameter also missing at installer level)? Shouldn't we do something like:
> 
> ```
> Installers:
> - Architecture: x86
>   InstallerUrl: https://win.desktop.evernote.com/builds/Evernote-10.101.6-win-ddl-stage-20240814195043-630c415c3da5ff7ae8f733a5c0d36a1691c06135-setup.exe
>   InstallerSha256: 871368AB87E5DAEFA014CB5DF6BB7B95FBB5BA7FC505102B6B23F6D90198F701
>   Scope: user
>   InstallerSwitches:
>     Custom: /currentuser
>   ProductCode: e4251011-875e-51f3-a464-121adaff5aaa
> - Architecture: x86
>   InstallerUrl: https://win.desktop.evernote.com/builds/Evernote-10.101.6-win-ddl-stage-20240814195043-630c415c3da5ff7ae8f733a5c0d36a1691c06135-setup.exe
>   InstallerSha256: 871368AB87E5DAEFA014CB5DF6BB7B95FBB5BA7FC505102B6B23F6D90198F701
>   Scope: machine
>   InstallerSwitches:
>     Custom: /allusers
>   ProductCode: e4251011-875e-51f3-a464-121adaff5aaa
>   ElevationRequirement: elevationRequired
> - Architecture: x64
>   InstallerUrl: https://win.desktop.evernote.com/builds/Evernote-10.101.6-win-ddl-stage-20240814195043-630c415c3da5ff7ae8f733a5c0d36a1691c06135-setup.exe
>   InstallerSha256: 871368AB87E5DAEFA014CB5DF6BB7B95FBB5BA7FC505102B6B23F6D90198F701
>   Scope: user
>   InstallerSwitches:
>     Custom: /currentuser
>   ProductCode: e4251011-875e-51f3-a464-121adaff5aaa
> - Architecture: x64
>   InstallerUrl: https://win.desktop.evernote.com/builds/Evernote-10.101.6-win-ddl-stage-20240814195043-630c415c3da5ff7ae8f733a5c0d36a1691c06135-setup.exe
>   InstallerSha256: 871368AB87E5DAEFA014CB5DF6BB7B95FBB5BA7FC505102B6B23F6D90198F701
>   Scope: machine
>   InstallerSwitches:
>     Custom: /allusers
>   ProductCode: e4251011-875e-51f3-a464-121adaff5aaa
>   ElevationRequirement: elevationRequired
> ```

A mistake😅. Fixed.

**Review by mdanish-kh — APPROVED** [review]

**Comment by mdanish-kh** [comment]
[Policy] Needs-Manual-Merge

**Comment by stephengillie** [comment]
This doesn't need manual merge, just a [conversation](https://github.com/microsoft/winget-pkgs/pull/167809#discussion_r1718941634) resolved.

**Pull request merged**